libs/application/templates/driving-assessment-approval/README.md

```diff
@@ -0,0 +1,76 @@
+# Reference Template
+
+This library is a reference how all application template libraries can be.
+
+## Requirements
+
+There are multiple requirements needed for a new template to be usable by the application system:
+
+1. Add a unique application type to `application/core/src/types/ApplicationType.ts`
+2. Build a new library similar to this one under the folder `application/templates/`
+3. The default export of this library has to be an object that extends the `ApplicationTemplate` interface
```

Comment:
This `README` seems to contain text from the reference template, consider updating or removing it